sources/vcdMerge/src/SignalSource.cpp

```diff
@@ -97,11 +99,28 @@ void MERGE::SignalSource::Create()
 
 void MERGE::SignalSource::SetFormat(const std::string &rFormat)
 {
-    if (rFormat != "T")
+    if (rFormat == "T")
     {
-        throw EXCEPTION::VcdException(EXCEPTION::Error::INVALID_LOG_FILE_FORMAT,
-                                      "Invalid log file format: " + rFormat);
+        m_pSignalFactory = std::make_unique<PARSER::SignalFactory>();
+        return;
     }
+
+    if (rFormat.length() > 3)
+    {
+        if ((rFormat[0] == 'U') &&
```

Comment:
Magic numbers.
(I mean 'U')

Reply:
This if could be: `if(IsUserXmlFormat(rFormat))`
The above if also could use some function for consistency.